packages/contracts/test/core/permission/permission-manager.ts

```diff
@@ -164,19 +167,38 @@ describe('Core: PermissionManager', function () {
   });
 
   describe('grantWithCondition', () => {
+    before(async () => {
+      conditionMock = await new PermissionConditionMock__factory(
+        signers[0]
+      ).deploy();
+    });
+
+    it('reverts if the condition address does not support `IPermissionCondition`', async () => {
```

Comment:
Misleading description. What the code checks for is whether the contract is an address or not. To be compilant with this, the codebase should check for the condition interface and not `isContract()`

Reply:
"...does not support `IPermissionCondition`" you mean: ... if the condition address is not a contract ?

Reply:
One thing is supporting `IPermissionCondition` and a different one is being a contract. 

The test says that it checks one thing but it effectively checks another one

Reply:
Changed in 21936b227091eccb877cc28b9544768e8d247234